driver/nbd_dispatch.c

```diff
@@ -234,7 +240,7 @@ NbdProcessDeviceThreadReplies(_In_ PWNBD_DISK_DEVICE Device)
     }
     KeReleaseSpinLock(&Device->SubmittedReqListLock, Irql);
     if(!Element) {
-        WNBD_LOG_ERROR("Received reply with no matching request tag: 0x%llx",
+        WNBD_LOG_WARN("Received reply with no matching request tag: 0x%llx",
```

Comment:
Let's use a debug message here. This will be printed for every abandoned request and there can be lots of them.

Reply:
Ughh, my bad. We must use WNBD_LOG_ERROR for nbd_dispatch since that will close the NBD connection. It's a non-fatal error for wnbd_dispatch, where I see that you're already using WNBD_LOG_TRACE (might be worth using WNBD_LOG_DEBUG for consistency).